go.mod

```diff
@@ -3,7 +3,9 @@ module mvdan.cc/garble
 go 1.19
 
 require (
+	github.com/bluekeyes/go-gitdiff v0.7.0
 	github.com/frankban/quicktest v1.14.3
+	github.com/gofrs/flock v0.8.1
```

Comment:
no need for an extra file here, we can use Go's locked file support :) https://pkg.go.dev/github.com/rogpeppe/go-internal@v1.9.0/lockedfile

as a bonus, I hope they will expose it upstream directly at some point, so the API is likely going to be similar: https://github.com/golang/go/issues/33974